### ISSUE TITLE
Rotated choices in choicegroup

### ISSUE THREAD

**Issue by dansom** [sdaps/sdaps#94]
Hi@all,

I'm working with SDAPS for a longer time now and most things work really lovely.

One thing I've been missing though is the ability to rotate the possible choices in a choicegroup, i.e. to vertically write these possible choices. Since an increasing amount of possible choices as well as long names of these choices decrease the space available for questions, this feature would be really great.

I also found a document online ([http://sdaps.org/archive/msg00441.html](http://sdaps.org/archive/msg00441.html)) that describes a new class / feature that allows exactly this (see the "new commands" section). Unfortunately the given example does not seem to work on my install, although it is 1.1.11.

When will this new feature be available and are there any workarounds till then?

Thanks in advance, Daniel

**Comment by dansom** [comment]
Just to add: Using the whole document from the website won't even work due to the missing documentclass "sdapsclassic". Just using the specific choicegroup with the rotated choices unfortunately also doesn't work :-/

**Comment by benzea** [comment]
Oh, sorry for not responding earlier.

This code is in development and is not yet supported in the normal SDAPS version. You need a specific development version (the "variables" branch). You also need the sdaps-class repository for the "sdapsclassic" class to exist.

See the PPS in http://sdaps.org/archive/msg00441.html.

**Comment by dansom** [comment]
Hi benzea,

thanks for the reply and the explanation. Since the message I found was in the archive, i somehow figured this would already be implemented in the main SDAPS version.

In the meantime I found a not so nice but useful workaround. I used latex to put white rectangles over original choices and then put another layer of text above them. That text is no longer within the choicegroup environment and therefore can be formatted however one wishes.

I will try the sdapsclassic class for the next project and see how it behaves :-)
